usecases/MinimumWorkingExample/dodo.py

```diff
@@ -57,7 +60,7 @@ def task_extract_metadata_emodul():
                 'actions': [(emodul_metadata, [raw_data_path, yaml_metadata_file])],
                 'file_dep': [raw_data_file],
                 'targets': [yaml_metadata_file],
-                'clean': [clean_targets]
+                'clean': [clean_targets]  # what does this do?
```

Comment:
remove comments

Reply:
I copied the version of the dodo file from main branch. this was also part of the merge problem.